demo/index.html

```diff
@@ -47,13 +47,13 @@
 <body>
 <div id="world-map" data-colors = '{"hover": "red", "highlighted":  "blue"   }'  data-countries=' {
 "AT":
-{"name": "Austria", "main_image": "src", "description": "brief description"}
+{"name": "Austria", "main_image": "src", "websites": "brief description"}
```

Comment:
shit I forget to delete D: old data is not using anymore in the example ... deleting...